.github/CODEOWNERS

```diff
@@ -64,7 +64,7 @@
 /sdk/iot/ @YoDaMa @xirzec
 
 # PRLabel: %KeyVault
-/sdk/keyvault/ @maorleger @sadasant 
+/sdk/keyvault/ @maorleger @sadasant @diberry
```

Comment:
This is fine but Just to be sure, this will include you as a reviewer for _every_ PR touching keyvault - including code changes, test changes, infra changes - across 4 packages. Is that what you want? Or are you specifically looking to be notified of any sample changes - maybe we can find a more specific pattern

Reply:
Just this sample but this is what Ramya suggested. If you have a better approach, I would prefer something less than all.

Reply:
The more reviewers, the better! 😆

Reply:
@diberry what all do you want to be added as a code reviewer for? You can definitely add another entry in the codeowners file to scope to just that file or directory.

Reply:
@weshaggard What would the syntax look like for a single file or directory or do you have a link to that information?

Reply:
General docs are at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners but as an example if you are only interested in this one sample you would do this:
```
sdk/keyvault/keyvault-certificates/samples-dev/helloWorld.ts  @maorleger @sadasant @diberry
```
I would keep the others there as well so they also still get notified as if you only add yourself you would override those settings the settings for this file.